src/System.IO.Compression/src/System/IO/Compression/ZipArchiveEntry.cs

```diff
@@ -924,9 +949,12 @@ private void WriteLocalFileHeaderAndDataIfNeeded()
                     WriteLocalFileHeader(false);
                     // we just want to copy the compressed bytes straight over, but the stream apis
                     // don't handle larger than 32 bit values, so we just wrap it in a stream
-                    using (MemoryStream compressedStream = new MemoryStream(_compressedBytes))
+                    foreach (Byte[] compressedBytes in _compressedBytes)
```

Comment:
I agree that comment didn't make sense.  I agree cleaning that up in a followup - thank you.